lib/fluent/log.rb

```diff
@@ -202,8 +229,12 @@ def time_format=(time_fmt)
       @time_formatter = Strftime.new(@time_format) rescue nil
     end
 
+    def stdout?
+      @out == $stdout
+    end
+
     def reopen!
-      @logger.reopen! if @logger
+      @out.reopen(@path, "a") if @path && @path != "-"
       nil
     end
```

Comment:
Note: The previous logic is meaningless.

This method was not used, and even if this is called, it will do nothing.

https://github.com/treasure-data/serverengine/blob/v2.3.2/lib/serverengine/daemon_logger.rb#L116-L119

https://github.com/treasure-data/serverengine/blob/v2.3.2/lib/serverengine/daemon_logger.rb#L43-L65

`filedev` is always `nil` here.